src/shared/views/MoreOptions.js

```diff
@@ -354,6 +370,10 @@ const mapStateToProps = (state, ownProps) => ({
     navigation: ownProps.navigation,
     profile: state.firebase.profile,
     progress: state.ui.user.progress,
+    teamId: state.ui.user.teamId,
+    teamName: state.firebase.data.teamDetails
+        ? state.firebase.data.teamDetails.teamName
```

Comment:
where does `teamDetails` come from?

Reply:
See my other comment a bit further down on this page.